doc/api/fs.md

```diff
@@ -7581,6 +7653,99 @@ added:
 * {number|bigint}
 
 Type of file system.
+This value is the decimal representation of the magic number* of the filesystem. Converting this number|bigint to hexadecimal will return the magic number.
+
+*magic number- refers to the hexadecimal numeric coding mapped to each filesystem.
+
+Here are some common filesystems and their magic numbers:
+<table>
+  <tr>
+    <th>Filesystem</th>
+    <th>Magic Number (Hexadecimal)</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>ext2</td>
+    <td>0xEF53</td>
+    <td>Second Extended File System (Linux)</td>
+  </tr>
+  <tr>
+    <td>ext3</td>
+    <td>0xEF53</td>
+    <td>Third Extended File System (Linux)</td>
+  </tr>
+  <tr>
+    <td>ext4</td>
+    <td>0xEF53</td>
+    <td>Fourth Extended File System (Linux)</td>
+  </tr>
+  <tr>
+    <td>Btrfs</td>
+    <td>0x9123683E</td>
+    <td>B-tree File System (Linux)</td>
+  </tr>
+  <tr>
+    <td>XFS</td>
+    <td>0x58465342</td>
+    <td>X File System (Linux)</td>
+  </tr>
+  <tr>
+    <td>FAT</td>
+    <td>0xadf5</td>
+    <td>File Allocation Table (legacy FAT)</td>
+  </tr>
+  <tr>
+    <td>NFS</td>
+    <td>0x6969</td>
+    <td>Network File System</td>
+  </tr>
+  <tr>
+    <td>ISO 9660</td>
+    <td>0x9660</td>
+    <td>CD-ROM File System</td>
+  </tr>
+  <tr>
+    <td>SMB/CIFS</td>
+    <td>0xFF534D42</td>
+    <td>Server Message Block / Common Internet File System</td>
+  </tr>
+  <tr>
+    <td>JFFS2</td>
+    <td>0x72B6</td>
+    <td>Journaling Flash File System</td>
+  </tr>
+  <tr>
+    <td>UDF</td>
+    <td>0x15013346</td>
+    <td>Universal Disk Format</td>
+  </tr>
+  <tr>
+    <td>procfs</td>
+    <td>0x9fa0</td>
+    <td>Process File System</td>
+  </tr>
+  <tr>
+    <td>sysfs</td>
+    <td>0x62656572</td>
+    <td>System File System</td>
+  </tr>
+  <tr>
+    <td>tmpfs</td>
+    <td>0x01021994</td>
+    <td>Temporary File System</td>
+  </tr>
+  <tr>
+    <td>cgroupfs</td>
+    <td>0x27e0eb</td>
+    <td>Control Group File System</td>
+  </tr>
+  <tr>
+    <td>ramfs</td>
+    <td>0x858458f6</td>
+    <td>RAM File System</td>
+  </tr>
+</table>
+
```

Comment:
Instead of a table, is there a wikipage or something that users can be referred to?

Reply:
I found a [Linux Manual page](https://man7.org/linux/man-pages/man2/statfs.2.html) which lists these filesystems but I am unsure whether we can link it here. Also, they only list Linux filesystems and miss out on the popular NTFS, NFS etc.